src/current/_includes/releases/v24.1/v24.1.4.md

```diff
@@ -0,0 +1,98 @@
+
+- Added an option to prevent the [`GRANT OPTION`]({% link v24.1/show-grants.md %}#privilege-grants) from being given to the owner of an object by default. The new `sql.auth.grant_option_for_owner.enabled` cluster setting defaults to `TRUE`, retaining the existing behavior, and when it is set to `FALSE`, the `GRANT OPTION` is not implicitly given to an object's owner. The owner still has all privileges on an object except the ability to grant privileges to other users. [#126959][#126959]
+
+<h3 id="v24-1-4-command-line-changes">Command-line changes</h3>
+
+- A `--locality-file` flag is now available on the [`cockroach start`]({% link v24.1/cockroach-start.md %}) and [`cockroach start-single-node`]({% link v24.1/cockroach-start-single-node.md %}) commands. This provides the option of specifing node [locality]({% link v24.1/cockroach-start.md %}#locality) (typically a `region` value) as a file, as an alternative to specifying this using the [`--locality` flag]({% link v24.1/cockroach-start.md %}#locality). [#127475][#127475]
+
+<h3 id="v24-1-4-db-console-changes">DB Console changes</h3>
+
+- The [databases]({% link v24.1/ui-databases-page.md %}) and [tables]({% link v24.1/ui-databases-page.md %}#tables-view) pages in the [DB Console]({% link v24.1/ui-overview.md %}) now show a loading state while loading information for databases and tables, including size and range counts. [#127748][#127748]
+- In the DB details page, the table name will no longer appear with quotes around the schema and table name. [#127766][#127766]
+
+<h3 id="v24-1-4-bug-fixes">Bug fixes</h3>
+
+- Fixed a bug introduced in v23.2.0 in which CockroachDB would hit an internal error when evaluating [`INSERT`s]({% link v24.1/insert.md %}) into [`REGIONAL BY ROW`]({% link v24.1/alter-table.md %}#set-the-table-locality-to-regional-by-row) tables where the source was a [`VALUES`]({% link v24.1/selection-queries.md %}#values-clause) clause with a single row and at least one boolean expression. [#127277][#127277]
+- Fixed a bug in which the [`DISCARD`]({% link v24.1/alter-table.md %}) statement was disallowed when the [session setting]({% link v24.1/session-variables.md %}#default-transaction-read-only) `default_transaction_read_only` was set to `on` [#127363][#127363]
+- In the [DB Console event log]({% link v24.1/ui-overview-dashboard.md %}#events-panel), [`ALTER ROLE`]({% link v24.1/alter-role.md %}) events now display correctly even when no [role options]({% link v24.1/alter-role.md %}#role-options) are included in the `ALTER ROLE` statement. [#126568][#126568]
+- Fixed a formatting issue with the `sql_sequence_cached_node` value of the `serial_normalization` [setting]({% link v24.1/session-variables.md %}). This could lead to an error connecting to CockroachDB if this value was set as the default for `serial_normalization` via cluster setting [`sql.defaults.serial_normalization`]({% link v24.1/cluster-settings.md %}#setting-sql-defaults-serial-normalization). [#127673][#127673]
```

Comment:
```suggestion
- Fixed a formatting issue with the `sql_sequence_cached_node` value of the [`serial_normalization` session setting]({% link v24.1/session-variables.md %}). This could lead to an error connecting to CockroachDB if this value was set as the default for `serial_normalization` via the cluster setting [`sql.defaults.serial_normalization`]({% link v24.1/cluster-settings.md %}#setting-sql-defaults-serial-normalization). [#127673][#127673]
```